frontend/src/components/MultiYearSummaryStatistics.vue

```diff
@@ -0,0 +1,168 @@
+<template>
```

Comment:
J'ai un petit "0" qui sort de la gauche sur une barre, pas sur pourquoi :thinking: 

Au passage un nice-to-have... est-ce qu'on peut ajouter de la transparence à l'indicateur de l'objectif EGAlim ?

![image](https://user-images.githubusercontent.com/1225929/129545069-02809ea7-68fd-4a87-a5c9-07b05adc1f36.png)

Reply:
Je le ne vois pas maintenant, est-ce que tu peux verifier encore ? Je sais pas si c'était reparer avec le changement en verticale.
![Screenshot 2021-08-16 at 17 23 39](https://user-images.githubusercontent.com/9282816/129588194-41b0bcbb-b509-4bfc-a0e5-2be0a1e3e3bb.png)

Reply:
L'`opacity` ici est pour un région (par exemple, si on veut souligner un région 50-55 sur le chart) mais il n'y a pas une option pour le label. J'ai pas réussi aussi avec `cssClass` sur le label, le class est ajouté sur le `text` et pas le `rect`
![Screenshot 2021-08-16 at 17 43 57](https://user-images.githubusercontent.com/9282816/129591297-90c98b5b-4332-41d6-b99f-677b5d53819c.png)